fronts-client/src/constants/feastPalettes.ts

```diff
@@ -801,6 +982,36 @@ export const feastCollectionPalettes: PaletteOption[] = [
 		},
 		imageURL: 'https://uploads.guim.co.uk/2024/03/21/TVsbacks.png',
 	},
+	{
+		id: 'utensils_1',
+		name: 'Utensils 1',
+		palettes: {
+			light: {
+				foregroundHex: '#C15627',
+				backgroundHex: '#F9F9F5',
+			},
+			dark: {
+				foregroundHex: '#F0CDB5',
+				backgroundHex: '#363632',
+			},
+		},
+		imageURL: 'https://uploads.guim.co.uk/2024/11/18/Utensils_1.png',
+	},
+	{
+		id: 'utensils_2',
+		name: 'Utensils 2',
```

Comment:
I think `Utensil 1` and `Utensil 2` are okay in terms of display names and distinguishing between two.
or let me know if we need just `Utensil`  for both?

Reply:
I agree 😁